package.json

```diff
@@ -4,16 +4,19 @@
   "private": true,
   "author": "Matthew Brandly",
   "license": "MIT",
+  "main": "src/angular-youtube-embed.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/brandly/angular-youtube-embed.git"
   },
   "homepage": "http://brandly.github.io/angular-youtube-embed/",
   "scripts": {
-    "postinstall": "bower install",
     "start": "gulp",
     "test": "gulp test"
   },
+  "dependencies": {
+    "angular": ""
```

Comment:
` peerDependencies` can be kinda flakey, and a blank version can serve the same purpose (if it's installed elsewhere, it will use that.) Again, feel free to do it as you like, but a lot of people are moving away from `peerDependencies`.